code/components/citizen-scripting-lua/src/LuaScriptRuntime.cpp

```diff
@@ -194,7 +194,7 @@ class LuaScriptRuntime : public OMClass<LuaScriptRuntime, IScriptRuntime, IScrip
 
 LuaScriptRuntime::~LuaScriptRuntime()
 {
-	

```

Comment:
Seriously? Fix your editor. :/

Reply:
Don't need fixing, it just try to be consistent, anyway readd them